orange/src/main/java/com/hartwig/hmftools/orange/algo/purple/ChromosomalRearrangementsDeterminer.java

```diff
@@ -0,0 +1,93 @@
+package com.hartwig.hmftools.orange.algo.purple;
+
+import java.util.List;
+
+import com.hartwig.hmftools.common.purple.PurpleCopyNumber;
+import com.hartwig.hmftools.common.genome.refgenome.RefGenomeCoordinates;
+import com.hartwig.hmftools.datamodel.orange.OrangeRefGenomeVersion;
+
+import org.jetbrains.annotations.NotNull;
+
+final class ChromosomalRearrangementsDeterminer
+{
+    private final RefGenomeCoordinates refGenomeCoordinates;
+
+    public ChromosomalRearrangementsDeterminer(@NotNull final OrangeRefGenomeVersion refGenomeVersion)
+    {
+        refGenomeCoordinates = getRefGenomeCoordinates(refGenomeVersion);
+    }
+
+    public static final double LOWER_THRESHOLD_TRISOMY = 2.8;
+    public static final double UPPER_THRESHOLD_TRISOMY = 3.5;
+    public static final double THRESHOLD_DELETION = 0.2;
+    public static final double PCT_98 = 0.98;
+    public static final double PCT_90 = 0.90;
+
+    private static final String CHR_1 = "1";
+    private static final String CHR_19 = "19";
```

Comment:
This logic won't work if the pipeline is run in v38 mode since the chromosomes will be prefixed with `chr` in that case. There are multiple ways to handle this, one way would be to always strip the prefix prior to comparison with your constants (see `RefGenomeFunctions.stripChrPrefix`)

Reply:
Thanks for catching this!